firmware/network/radio/radio.c

```diff
@@ -83,35 +83,35 @@ int8_t get_ieee802154_iface(void) {
     return -1;
 }
 
-int8_t get_netopt_tx_power(int16_t txpower) {
+int8_t get_netopt_tx_power(int16_t *txpower) {
     int res;
     int index = get_ieee802154_iface();
     if (index < 0) {
         return -1;
     }
     netif_t *iface = netif_get_by_id(index);
 
-    res = netif_get_opt(iface, NETOPT_TX_POWER, 0, &txpower, sizeof(txpower));
+    res = netif_get_opt(iface, NETOPT_TX_POWER, 0, txpower, sizeof(int16_t));
```

Comment:
What do you think about this proposal?
`int res = netif_get_opt(iface, NETOPT_TX_POWER, 0, txpower, sizeof(int16_t));`
Because it's defined on line 87 but not used until the line 94